vcd/resource_vcd_nsxt_network_dhcp.go

```diff
@@ -177,32 +177,37 @@ func resourceVcdOpenApiDhcpDelete(ctx context.Context, d *schema.ResourceData, m
 func resourceVcdOpenApiDhcpImport(ctx context.Context, d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
 	resourceURI := strings.Split(d.Id(), ImportSeparator)
 	if len(resourceURI) != 3 {
-		return nil, fmt.Errorf("resource name must be specified as org-name.vdc-name.org_network_name")
+		return nil, fmt.Errorf("resource name must be specified as org-name.vdc-org-vdc-group-name.org_network_name")
 	}
-	orgName, vdcName, orgVdcNetworkName := resourceURI[0], resourceURI[1], resourceURI[2]
+	orgName, vdcOrVdcGroupName, orgVdcNetworkName := resourceURI[0], resourceURI[1], resourceURI[2]
 
 	vcdClient := meta.(*VCDClient)
-	org, err := vcdClient.GetAdminOrg(orgName)
-	if err != nil {
-		return nil, fmt.Errorf("unable to find Org %s: %s", orgName, err)
-	}
-	vdc, err := org.GetVDCByName(vdcName, false)
-	if err != nil {
-		return nil, fmt.Errorf("unable to find VDC %s: %s", vdcName, err)
+	// define an interface type to match VDC and VDC Groups
+	var vdcOrVdcGroup vdcOrVdcGroupHandler
+	_, vdcOrVdcGroup, err := vcdClient.GetOrgAndVdc(orgName, vdcOrVdcGroupName)
```

Comment:
generic error handling missing

Reply:
Yes! I just forgot to port the newer solution!

Reply:
Tests worked:
```
== RUN   TestAccVcdOpenApiDhcpNsxtRoutedDS
--- PASS: TestAccVcdOpenApiDhcpNsxtRoutedDS (48.61s)
=== RUN   TestAccVcdOpenApiDhcpNsxtRouted
--- PASS: TestAccVcdOpenApiDhcpNsxtRouted (72.34s)
```